src/components/Table/GridTable.tsx

```diff
@@ -356,14 +356,14 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
   // Make a single copy of our current collapsed state, so we'll have a single observer.
   const collapsedIds = useComputed(() => rowState.collapsedIds, [rowState]);
 
-  const [sortState, setSortKey] = useSortState<R, S>(columns, sorting);
+  const [sortState, setSortKey, sortOn] = useSortState<R, S>(columns, props.sorting);
```

Comment:
Most places that were depending on `sorting` really just needed `sortOn`, which is a primitive of `"client" | "server" | undefined`.